commands/local_server_start.go

```diff
@@ -169,6 +174,17 @@ var localServerStartCmd = &console.Command{
 			}
 		}
 
+		// If 'SSLKEYLOGFILE' environment variable is set, uses this as a destination of TLS key log.
+		// In this context, the name 'SSLKEYLOGFILE' is common, so using 'SSL' instead of 'TLS' name.
+		// This environment variable is preferred than the key log file from the console argument.
+		if path := os.Getenv("SSLKEYLOGFILE"); path != "" {
```

Comment:
it's possible to let `console` take care of this by specifying `EnvVars` on the associated flags.
See #268